### PR TITLE
Revert "force omission of meta files (#2662)"

### PR DESCRIPTION
**Pull request by meain**
This reverts commit 1945db6b65827e6dd6d57de8ac42fe4a119db685.

We do add the `isMeta` flags for missing items at https://github.com/alcionai/corso/blob/f60207ee32811a475d7803b9aa628a9fe80cff8c/src/pkg/repository/repository.go#L363

---

#### Does this PR need a docs update or release note?

- [ ] :white_check_mark: Yes, it's included
- [ ] :clock1: Yes, but in a later PR
- [x] :no_entry: No

#### Type of change

- [ ] :sunflower: Feature
- [ ] :bug: Bugfix
- [ ] :world_map: Documentation
- [ ] :robot: Test
- [ ] :computer: CI/Deployment
- [x] :broom: Tech Debt/Cleanup

#### Issue(s)

* #<issue>

#### Test Plan

- [ ] :muscle: Manual
- [x] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

**Comment by meain** [comment]
> I'm curious what the long-term plan for this is. Is this so that we can restore permissions for empty folders right now? Eventually we will want to remove the metadata files from backup details I think

I think we are good with removing metadata files from backup details as done [here](https://github.com/alcionai/corso/commit/83fe83a36728bd87c28ed40051906e62759357fe). This is mostly to make sure we can filter it out from older backups which does have metadata within them.

**Comment by ryanfkeepers** [comment]
@meain I'm worried about the potential for this to cascade to ark failures (again).  Not all entries will contain the isMeta flag.  I think I took out a version-based filter to ensure that older files were also successfully removed, last time I made this change.  If you brought back that check as well I'd feel a bit more secure here.

**Comment by meain** [comment]
@ryanfkeepers Unless I am missing something, the following code should add `IsMeta` to older versions as well.
https://github.com/alcionai/corso/blob/f60207ee32811a475d7803b9aa628a9fe80cff8c/src/pkg/repository/repository.go#L363-L372

**Comment by ashmrtn** [comment]
>FYI, this change does not revert omitting meta files. It just reverts filtering it twice.

Ah, sorry I missed that. I guess that's what I get for reading the title and skimming the code 😅

File: src/pkg/backup/details/details.go [hunk 1]
```diff
@@ -128,13 +127,7 @@ func (dm DetailsModel) FilterMetaFiles() DetailsModel {
...
 // be treated as regular files.
 func (de DetailsEntry) isMetaFile() bool {
-	if de.ItemInfo.OneDrive == nil {
-		return false
-	}
-
-	return de.ItemInfo.OneDrive.IsMeta ||
-		strings.HasSuffix(de.RepoRef, ".meta") ||
-		strings.HasSuffix(de.RepoRef, ".dirmeta")
+	return de.ItemInfo.OneDrive != nil && de.ItemInfo.OneDrive.IsMeta
```
Comment (ryanfkeepers) [inline src/pkg/backup/details/details.go:130]: Can you add a TODO to add sharepoint recognition here as well, please?

**Review by ryanfkeepers — APPROVED** [review]

**Pull request merged**